.travis.yml

```diff
@@ -19,7 +19,8 @@ script:
 - make cov-ci-run
 
 after_success:
-- codecov
+ # - codecov
+ - bash <(curl -s https://codecov.io/bash)
```

Comment:
why do this? we install it in CI pinned to certain version:
https://github.com/aio-libs/aiohttp/blob/master/requirements/ci-wheel.txt#L25

Reply:
It was my try to fix coverage reports.
Now upload on codecov is failed (with both codecov plugin and bash script).
Any ideas?